modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -588,6 +588,11 @@ const questionnaireStyle = theme => ({
     specialOptionSwitch: {
       margin: "0",
     },
+    optionsDragIndicator: {
+      float: "left",
+      padding: "12px 4px",
+      borderRadius: "4px"
```

Comment:
Use `theme.spacing`:
```suggestion
      padding: theme.spacing(1.5, 0.5),
      borderRadius: theme.spacing(0.5),
```